src/management-system-v2/app/(dashboard)/[environmentId]/executions/deployment-selection-icon-view.tsx

```diff
@@ -0,0 +1,70 @@
+import ScrollBar from '@/components/scrollbar';
+import ElementIconView, { ItemIconViewProps } from '@/components/item-icon-view';
+import { ProcessListProcess } from './deployments-modal';
+import { Button, Card } from 'antd';
+import { DraggableElementGenerator } from '@/components/processes/draggable-element';
+import Viewer from '@/components/bpmn-viewer';
+import { FolderOutlined } from '@ant-design/icons';
+
+const DraggableDiv = DraggableElementGenerator('div', 'itemId');
```

Comment:
In order for this to work, the components need to be wrapped in `DraggableContext`. But if you don't want the items to be draggable, you can just remove the Wrapper prop .